.github/workflows/gradle-latest-versions.yml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Gradle latest versions tests
+
+on:
+  schedule:
+    - cron: "0 7 * * MON"
+  push:
+    branches:
+      - "gradleLatestVersionsTestManualTrigger"
+      - "szpak/gradleLatestVersionsGHActions"
```

Comment:
The first branch is to be able to retrigger it on demand. The second will be removed before merge.

Do we need any "other" mode of execution?

Reply:
I guess "manual" trigger should better be configured via `workflow_dispatch`.
Then you have a GitHub UI for triggering a run, and you can even have parameters (e.g. a Gradle version to test): https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
It makes sense. I've been doing something similar with the variables in GitLab.

Reply:
I added `on.workflow_dispatch` which should be enough, but it's not :-).

![image](https://user-images.githubusercontent.com/148013/219951693-6f133c8e-680c-41b7-8121-a20e9f9ecc99.png)

In GitLab, I could select the branch to run workflow with, here, I don't see anything similar. On the main branch, there isn't "workflow_dispatch", maybe [because of that](https://stackoverflow.com/a/65389878) the GH Actions don't allow mi to run it manually... (but will, after it is merged into master). I don't have also GH cli configured to trigger it manually.

Reply:
Having it merged into master, there is a button:
![image](https://user-images.githubusercontent.com/148013/219975464-bbb79e8e-5fec-46a2-815b-66d4abdc2410.png)